scalafix/input/src/main/scala/fix/regexgroups.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ rule = SortImports
+ SortImports.blocks = [
+ "(java\\.|com\\.sun)",
```

Comment:
so here it's getting tricky
if we just say `(java|com\\.sun)` then any import starting with `java` (including javax) will match, so to get `desired` order, we have to say `java\\.` to differentiate from `javax` which is part of the second group regex